### ISSUE TITLE
Does DCO loss apply to general finetuning?

### ISSUE THREAD

**Issue by fffyanggg** [kyungmnlee/dco#2]
First, thanks for the great work. I really like it. I do have some questions about its use case: 
- will  DCO work with general finetuning, including LoRA and full unet finetuning? If so, do I just simply do weighted sum of regular diffusion training loss and DCO loss?
- how important is also tuning text encoder? If I just do full unet finetuning, do I still have to finetune text encoder as well? This could be very expensive.

Thanks and looking forward to your insights.

**Comment by kyungmnlee** [comment]
Thank you for your interest in our work! Regarding your questions:
1. I believe DCO will work with general fine-tuning (both LoRA or full fine-tuning). You can simply replace the regular diffusion training loss with DCO loss. Here, you could search for some hyperparameter beta.
2. I didn't experiment with tuning text encoder, yet we fine-tune the textual embedding (i.e. textual inversion). Tuning text encoder might help, but I believe fine-tuning unet (or LoRA) would suffice for many cases. 

Please let me know if you have further questions.
